src/PrestaShopBundle/Install/Simplexml.php

```diff
@@ -28,7 +28,7 @@
 
 use DOMDocument;
 
-class SimplexmlElement extends \SimpleXMLElement
+class Simplexml extends \SimpleXMLElement
```

Comment:
Hum, changing this class can break something :/ More if this classe is not loaded with the autoloader.